README.md

```diff
@@ -119,6 +119,62 @@ ClientConfig = [{reconnect_cool_down_seconds, 10}],
 ok = brod:start_client([{"localhost", 9092}], brod_client_1, ClientConfig).
 ```
 
+## Handling of connection errors
```

Comment:
This section looks like consumer specific. perhaps move it down to the `Consumers` section after Group Subscriber ?